src/cargo/ops/registry.rs

```diff
@@ -1002,49 +1004,117 @@ pub fn modify_owners(config: &Config, opts: &OwnersOptions) -> CargoResult<()> {
         Some(mutation),
     )?;
 
-    if let Some(ref v) = opts.to_add {
-        let v = v.iter().map(|s| &s[..]).collect::<Vec<_>>();
-        let msg = registry.add_owners(&name, &v).with_context(|| {
-            format!(
-                "failed to invite owners to crate `{}` on registry at {}",
-                name,
-                registry.host()
-            )
-        })?;
+    if opts.subcommand == Some("".to_string()) {
```

Comment:
Comparing against sentinel values, like `""`, is a code smell.  In particular, here you are using an `Option` that will never be `None`.  Instead this case should be the `None` case and should be matched in the `match` statement below.

Reply:
thank you.